catalog-info.yaml

```
@@ -0,0 +1,55 @@
+# Declare a Backstage Component that represents your application.
+---
+# yaml-language-server: $schema=https://json.schemastore.org/catalog-info.json
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: stream
+  description: Test utility for streaming data via udp/tcp/tls/webhook/GCP Pub-Sub/Kafka.
+
+spec:
+  type: library
```

looks like more a CLI tool, no? 

```suggestion
  type: tool
```